features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -360,7 +360,7 @@ class ApiConsole extends React.Component {
                 <Grid container className={classes.grid}>
                     {!user && (
                         <Grid item md={6}>
-                            <Paper className={classes.userNotificationPaper}>
+                            <Paper elevation={2} className={classes.userNotificationPaper}>
```

Comment:
Do we need to set `elevation={2}` specifically here ?

Reply:
according to [1] the default elevation in v3 is similar to elevation=2 in v4

[1] - https://material-ui.com/guides/migration-v3/